src/module.ts

```diff
@@ -128,6 +135,18 @@ export default defineNuxtModule<NuxtI18nOptions>({
     applyLayerOptions(options, nuxt)
     await mergeI18nModules(options, nuxt)
 
+    /**
+     * source baseUrl from Nuxt SEO's `site.url`
+     * TODO: remove i18n's `baseUrl` module option when `site.url` has been in use for a sufficiently long time (starting October 2023)
+     */
+
+    await installNuxtSiteConfig()
+    const siteConfig = useSiteConfig()
+
+    if (siteConfig.url) {
+      options.baseUrl = siteConfig.url
```

Comment:
Is this needed? We want to use set the baseUrl in a runtime environment so that it can be easily switched

Reply:
Ah fair. Yeah, currently I only set it right here at build time, that should change.

Reply:
In ea4a900dba9f48452360dbe8668cb37f07cd0d7e I made that change. There could be a better place of course as @BobbieGoede pointed out, but I don't feel knowledgeable enough to decide on that.